web/templates/draft.hbs

```diff
@@ -35,6 +35,7 @@
                 <div class="messagebox-content">
                     <div class="message_top_line">
                         <div class="overlay_message_controls">
+                            <i class="fa fa-copy fa-lg copy-overlay-message tippy-zulip-tooltip" aria-hidden="true" data-tippy-content="Copy draft"></i>
```

Comment:
I have used the translation string, but I don't think using a template here would make sense because we generally use a template for strings that has shortcuts to show, but here we just have a static string.

Reply:
Yeah, that also makes sense. :+1: